src/paths/reviews.yaml

```diff
@@ -0,0 +1,59 @@
+#get reviews from entry
+get:
+  tags:
+    - reviews
+  description: "Returns all reviews of an entry"
+  summary: "/reviews"
+  parameters:
+    - $ref: "../parameters/query/limit.yaml"
+    - $ref: "../parameters/query/after.yaml"
+    - $ref: "../parameters/path/entry_id.yaml"
```

Comment:
A small thing I like to make clear. You cannot have a path parameter (`entry_id` in this case) that is not represented in the path (`/reviews` here).

Reply:
Sorry, i didn't know!
My bad